### PR TITLE
QCElemental Integration

### PR DESCRIPTION
**Pull request by dgasmith**
We have added a specific `to_string` option for nglview which provides a higher quality interface and reduces dependancies. I have also corrected the name to reflect our common case.

In the docs you name this function `show_qcelementary`, but I couldn't find where to edit them.

### CODE REVIEW

File: nglview/show.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def show_iodata(obj, **kwargs):
 
 
 def show_qcelemental(obj, **kwargs):
-    """Show QCelemental's Molecule (require `ase` package).
+    """Show QCElemental's Molecule (require `ase` package).
```
Comment (hainm) [inline nglview/show.py]: remove `(require `ase` package).`?

**Comment by hainm** [comment]
thanks @dgasmith. I have only 1 minor comment and we're good to merge.

**Comment by dgasmith** [comment]
Fixed, thanks for the review.

**Comment by hainm** [comment]
Thank you.

**Pull request merged**